tool/tctl/common/acl_command.go

```diff
@@ -0,0 +1,251 @@
+	c.usersAdd.Arg("expires", "When the user's access expires (must be in RFC3339).").Required().StringVar(&c.expires)
+	c.usersAdd.Arg("reason", "The reason the user has been added to the access list.").Required().StringVar(&c.reason)
+
+	c.usersRemove = users.Command("rm", "Remove a user from an access list.")
+	c.usersRemove.Arg("access-list-name", "The access list name.").Required().StringVar(&c.accessListName)
+	c.usersRemove.Arg("user", "The user to remove from the access list.").Required().StringVar(&c.userName)
+
+	c.usersList = users.Command("ls", "List users that are members of an access list.")
+	c.usersList.Arg("access-list-name", "The access list name.").Required().StringVar(&c.accessListName)
+}
+
+// TryRun takes the CLI command as an argument (like "acl ls") and executes it.
+func (c *ACLCommand) TryRun(ctx context.Context, cmd string, client auth.ClientI) (match bool, err error) {
+	switch cmd {
+	case c.ls.FullCommand():
+		err = c.List(ctx, client)
+	case c.ls.FullCommand():
+		err = c.Get(ctx, client)
+	case c.usersAdd.FullCommand():
+		err = c.UsersAdd(ctx, client)
+	case c.usersRemove.FullCommand():
+		err = c.UsersRemove(ctx, client)
+	case c.usersList.FullCommand():
+		err = c.UsersList(ctx, client)
+	default:
+		return false, nil
+	}
+	return true, trace.Wrap(err)
+}
+
+// List will list access lists visible to the user.
+func (c *ACLCommand) List(ctx context.Context, client auth.ClientI) error {
+	var accessLists []*accesslist.AccessList
+	var nextKey string
+	for {
+		var page []*accesslist.AccessList
+		var err error
+		page, nextKey, err = client.AccessListClient().ListAccessLists(ctx, 0, nextKey)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+
+		accessLists = append(accessLists, page...)
+
+		if nextKey == "" {
+			break
+		}
+	}
+
+	if len(accessLists) == 0 {
+		fmt.Println("no access lists")
+		return nil
+	}
+
+	return trace.Wrap(displayAccessLists(c.format, accessLists...))
+}
+
+// Get will display information about an access list visible to the user.
+func (c *ACLCommand) Get(ctx context.Context, client auth.ClientI) error {
+	accessList, err := client.AccessListClient().GetAccessList(ctx, c.accessListName)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	return trace.Wrap(displayAccessLists(c.format, accessList))
+}
+
+// UsersAdd will add a user to an access list.
+func (c *ACLCommand) UsersAdd(ctx context.Context, client auth.ClientI) error {
+	expires, err := time.Parse(time.RFC3339, c.expires)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	accessList, err := client.AccessListClient().GetAccessList(ctx, c.accessListName)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	accessList.Spec.Members = append(accessList.Spec.Members, accesslist.Member{
+		Name:    c.userName,
+		Reason:  c.reason,
+		Expires: expires,
+
+		// The following fields will be updated in the backend, so their values here don't matter.
+		Joined:  time.Now(),
+		AddedBy: "dummy",
```

Comment:
What's "dummy"?

Reply:
"dummy" is just a string to pass `CheckAndSetDefaults`. It will be overwritten in the backend.